minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterControl.java

```diff
@@ -153,17 +154,32 @@ public synchronized void startCoordinator(Class<? extends CompactionCoordinator>
       coordinatorProcess = cluster
           ._exec(coordinator, ServerType.COMPACTION_COORDINATOR, new HashMap<>()).getProcess();
       // Wait for coordinator to start
```

Comment:
So this no longer waits for the coordinator to start, it only waits a certain amount of time. Is that right?